### PR TITLE
docs: more detailed update Tabs api

### PR DESCRIPTION
**Pull request by jesse3mh9a**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by xrkffgg** [comment]
补充 https://github.com/ant-design/ant-design/pull/27410

**Review by xrkffgg — APPROVED** [review]

File: components/tabs/index.zh-CN.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Ant Design 依次提供了三级选项卡，分别用于不同的场景。
 | keyboard | 开启键盘切换功能 | boolean | true |  |
 | renderTabBar | 替换 TabBar，用于二次封装标签头 | (props: DefaultTabBarProps, DefaultTabBar: React.ComponentClass) => React.ReactElement | - |  |
 | size | 大小，提供 `large` `default` 和 `small` 三种大小 | string | `default` |  |
-| tabBarExtraContent | tab bar 上额外的元素 | ReactNode \| {left?: ReactNode, right?: ReactNode} | - | 4.6.0 |
+| tabBarExtraContent | tab bar 上额外的元素 | ReactNode \| {left?: ReactNode, right?: ReactNode} | - | object: 4.6.0 |
```
Comment (hengkx) [inline components/tabs/index.zh-CN.md:37]: 这个object 指什么 完全看不出来

Comment (xrkffgg) [reply components/tabs/index.zh-CN.md:37]: 是的

Comment (jesse3mh9a) [reply components/tabs/index.zh-CN.md:37]: ReactNode \| `*` {left?: ReactNode, right?: ReactNode}  

`*`: 4.6.0

or

1. ReactNode \| 2. {left?: ReactNode, right?: ReactNode}  

`2`: 4.6.0

Comment (xrkffgg) [reply components/tabs/index.zh-CN.md:37]: 保持现在。

Comment (jesse3mh9a) [reply components/tabs/index.zh-CN.md:37]: ok

**Pull request merged**